lib/console_web/controllers/router/device_controller.ex

```diff
@@ -170,25 +172,34 @@ defmodule ConsoleWeb.Router.DeviceController do
             Events.create_event(Map.put(event, "organization_id", organization.id))
           end)
           |> Ecto.Multi.run(:device, fn _repo, %{ event: event } ->
+            dc_used = 
+              case event.sub_category in ["uplink_confirmed", "uplink_unconfirmed"] do
```

Comment:
so only uplink confirmed or unconfirmed will drain DC and it is only possible to get this sub-category for each uplink once per uuid right?

Reply:
Yep, that's the assumption here. When I spoke to Pierre, he said `dc` will only come in with `uplink_confirmed` and `uplink_unconfirmed`.